### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by tdrwenski**
Initial setup for thredds performance regression tests. See readme for more info on how to run and configure new test cases.

### CODE REVIEW

**Comment by tdrwenski** [comment]
As Jen mentioned, it is nice if we can get something merged so we can test it on our server once that is set up.

**Review by oxelson — APPROVED** [review]

**Comment by oxelson** [comment]
Thanks for doing this @tdrwenski!    Looks good!

File: .gitignore
Comment (oxelson) [inline .gitignore:1]: What's the change?

Comment (tdrwenski) [reply .gitignore:1]: there was no newline at the end of the file so I added that now :)

**Review by oxelson — APPROVED** [review]

**Comment by haileyajohnson** [comment]
🎉

**Pull request merged**